### PR TITLE
implement navbar

### PR DESCRIPTION
**Pull request by abygeorge7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        </div>
+        <div class="navbar-end">
+            <div class="search-icon-container">
+                <img src="./images/search.svg" class="small-icon" alt="">
+            </div>
+            <h1 class="navbar-dropdown">
+                Sign in <img class="small-icon arrow-container" src="./images/chevron-down.svg" alt="">
+            </h1>
+            <button class="free-trial-button">TRY FOR FREE</button>
+            <div class="hamberger-icon-mobile">
```
Comment (BenjaminSao) [inline index.html]: hamburger*

File: images/search-copied.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 797 801" width="20" height="20" x="0px" y="0px">
```
Comment (BenjaminSao) [inline images/search-copied.svg:1]: You aren't using this svg. Why is it here?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-start">
+            <div class="main-logo">
+                <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/global/header/PS_logo.png"
+                    class="main-logo-icon" alt="main-logo">
+            </div>
+            <div class="navbar-items">
```
Comment (BenjaminSao) [inline index.html]: I names I provided you might not be the best. Maybe you could use better names for the classes such as this one. (e.g. "navbar-start-items" to denote that this is the starting container of the navbar)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                <h1 class="navbar-dropdown">
```
Comment (BenjaminSao) [inline index.html]: Here too, use maybe like navbar-dropdown-item or navbar-item-dropdown.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,103 @@
+.navbar {
```
Comment (BenjaminSao) [inline style.css]: You should have brackets on new line. 
(e.g. 
`.class \n
{ \n
... \n
}`) "\n" denote new lines

File: index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Spartan:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-start">
+            <div class="main-logo">
```
Comment (BenjaminSao) [inline index.html]: Maybe the class should be called navbar-brand-container or navbar-logo-container

File: style.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+  display: flex;
+  align-items: center;
+  padding-bottom: 2px;
+}
+
+body {
+  margin: 0;
+}
+
+h1 {
```
Comment (BenjaminSao) [inline style.css]: A good practice would be do declare all your h1, h2, h3,... to reuse text format.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+.small-icon {
+  width: 18px;
+  height: 18px;
+}
+
+.arrow-container {
+  margin-left: 4px;
+}
+
+.free-trial-button {
```
Comment (BenjaminSao) [inline style.css]: This button is also used elsewhere in the website, so you should name it "navbar-free-trial-button"

**Review by BenjaminSao — CHANGES_REQUESTED** [review]

File: style.css (hunk 5 above)
Comment (abygeorge7) [reply style.css]: do i need to set-up  my vs code for that.\n seems not working

File: style.css (hunk 7 above)
Comment (abygeorge7) [reply style.css]: as i only used h1 until now,do i need to declare the rest of items?

File: style.css (hunk 5 above)
Comment (BenjaminSao) [reply style.css]: "\n" is just a new line. If you see a \n it means press Enter on your keyboard

File: style.css (hunk 7 above)
Comment (BenjaminSao) [reply style.css]: True, you can do them later.

**Review by BenjaminSao — APPROVED** [review]

**Pull request merged**